doc/classes/RigidDynamicBody2D.xml

```diff
@@ -100,10 +100,17 @@
 		<method name="get_colliding_bodies" qualifiers="const">
 			<return type="Node2D[]" />
 			<description>
-				Returns a list of the bodies colliding with this one. Requires [member contact_monitor] to be set to [code]true[/code] and [member contacts_reported] to be set high enough to detect all the collisions.
+				Returns a list of the bodies colliding with this one. Requires [member contact_monitor] to be set to [code]true[/code] and [member max_contacts_reported] to be set high enough to detect all the collisions.
 				[b]Note:[/b] The result of this test is not immediate after moving objects. For performance, list of collisions is updated once per frame and before the physics step. Consider using signals instead.
 			</description>
 		</method>
+		<method name="get_contact_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the number of contacts this body has with other bodies. By default, this returns 0 unless bodies are configured to monitor contacts (see [member contact_monitor]).
+				[b]Note:[/b] to retrieve the colliding bodies, use [method get_colliding_bodies]
```

Comment:
```suggestion
				[b]Note:[/b] to retrieve the colliding bodies, use [method get_colliding_bodies].
```

Reply:
Second nitpick: the sentence after "Note:" should start with an capitalized word.